ci/scripts/windows/test-run.sh

```diff
@@ -70,12 +70,14 @@ pushd ${GEODE_BUILD}
   echo "Running tests"
   set -x
 
+  SCM_PROPS="-PsourceRevision=\"$(git rev-parse HEAD)\" -PsourceRepository=\"${SOURCE_REPOSITORY}\""
```

Comment:
We don't need to set these on Windows or any of the other test scripts since we don't ship the tested binaries.